code/game/machinery/doors/brigdoors.dm

```diff
@@ -277,6 +309,7 @@
 				F.flash()
 
 		if(href_list["change"])
+			printed = 1

```

Comment:
The reason there is a printed = 1, is because when you adjust time and hit change, it will run the timer_start again, but with the new time. This means that users would be prompted for input and charges and print a whole new paper everytime the time is adjusted. This printed = 1, stops the print proc from running.